simplq/src/styles/homePage.module.scss

```diff
@@ -20,22 +20,36 @@ div {
 .center {
   @include center-horizontally();
   @include text-color();
-  font-size: 1.5rem;
+  font-size: 26pt;
 }
 
 .benefits-info {
   @include background-color();
   @include text-color();
   @include center-vertically();
+  @include center-horizontally();
   div {
+    padding: 70px;
     margin: auto;
+    font-size: 23pt;
+    img {
+      height: 70px;
+    }
   }
 }
 
 .create-queue {
   @include text-color();
   @include center-vertically();
   @include center-horizontally();
+  div {
+    img {
+      height: 250px;
+    }
+    div {
+      padding: 70px;
+    }
+  }
 }
 
```

Comment:
Could you replace pt and px units with rem? I went with rem everywhere because rem is a relative unit that changes according to the parent's font size are recommended according to a stackoverflow answer or a blog I read.

Reply:
done